avakas/cli.py

```diff
@@ -85,7 +77,15 @@ def cli_bump_version(**kwargs):
 
     bump = kwargs['level'][0]
 
-    if not project.bump(bump=bump):
+    if prerelease_date is True:
```

Comment:
Code to add `prerelease_date` in `add_metadata` was moved to `cli_bump_version`.

`add_metadata` was called from there **and** `cli_set_version`, so `cli_set_version` will also need to be updated to add `prerelease_date` if we want to expect that behaviour will be consistent in that function from before the patch.

Reply:
This has been addressed, though I think we might want to wait for #74 to be merged in first (fixes #72, (insufficient test coverage) which is why this slipped by in the first place). I've created a little side branch and done the testing but I'd feel more confident that way